clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -73,13 +73,18 @@ def CastOp : CIR_Op<"cast", [Pure]> {
   let description = [{
     Apply C/C++ usual conversions rules between values. Currently supported kinds:
 
-    - `int_to_bool`
-    - `ptr_to_bool`
     - `array_to_ptrdecay`
-    - `integral`
     - `bitcast`
+    - `integral`
+    - `int_to_bool`
+    - `int_to_float`
     - `floating`
     - `float_to_int`
+    - `float_to_bool`
+    - `ptr_to_int`
+    - `ptr_to_bool`
+    - `bool_to_int`
+    - `bool_to_float`
```

Comment:
Seems unrelated to this patch? Probably best on its own PR next time.

Reply:
Okay.  This change is small enough that if I didn't make the change right away, I would probably never come back to it.  And it is documentation only, no behavioral change.